core/controllers/base_test.py

```diff
@@ -180,27 +180,24 @@ def test_requests_for_invalid_paths(self):
         user_id = user_services.get_user_id_from_username('learneruser')
         csrf_token = base.CsrfTokenManager.create_csrf_token(user_id)
 
-        self.get_html_response(
-            '/library/extra', expected_status_int=404)
-
         self.get_html_response(
             '/library/data/extra', expected_status_int=404)
 
         self.post_json(
-            '/library/extra', payload={}, csrf_token=csrf_token,
+            '/library/data/extra', payload={}, csrf_token=csrf_token,
```

Comment:
These changes had to be made, since the new topic landing page handlers accept URLs in the form of `/<>/<>` so `/library/extra` would get matched with the topic landing page handler and not be recognized as an invalid path.

Reply:
That sounds like the `/<>/<>` pattern is too broad. Could the first part of the prefix at least be hard-coded or added to the urls list in a similar way to the classroom pages?